astropy/time/tests/test_methods.py

```diff
@@ -620,6 +621,35 @@ def test_sort(self, use_mask):
             assert np.all(self.t0.sort(-1)[:, :, 0] == self.t0.min(-1))
             assert np.all(self.t0.sort(-1)[:, :, -1] == self.t0.max(-1))
 
+    def test_mean(self, use_mask):
+        self.create_data(use_mask)
+
+        if use_mask == 'not_masked':
+            assert self.t0.mean().to_value('jd') == self.t0.to_value('jd').mean()
+            assert self.t1.mean().to_value('jd') == self.t1.to_value('jd').mean()
+
+            epsilon = 1 * u.ns
+
+            t0 = Time('2021-07-27T00:00:00')
+            t1 = Time('2022-07-27T00:00:00')
+            t2 = Time('2023-07-27T00:00:00')
+
+            t0.format = 'jd'
```

Comment:
Why does the format matter for the test?

Reply:
I suppose it doesn't. For some reason I was under the impression that the 'isot' format didn't have enough precision to capture the 1 ns change, but after testing I can see it doesn't matter.

Reply:
The format is only important for displaying, the internal representation is always the same (the two jd floats)